platforms/core-runtime/base-services/src/main/java/org/gradle/util/Path.java

```diff
@@ -118,7 +118,18 @@ public Path append(Path path) {
         return new Path(concat, absolute);
     }
 
+    @Deprecated
     public String getPath() {
+        return asString();
```

Comment:
❓ If we want to later remove this method, we should also do a deprecation logging here. See `DeprecationLogger` usage in other places for this.